compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -832,6 +832,9 @@ error.ambiguous.type=\
 error.uninitialized.variable=\
   variable ''{0}'' is not initialized
 
+error.uninitialized.fields=\
+  all fields must be initialized
```

Comment:
Shall we improve the error message? One improvement we can do here is, print all uninitialized fields.

Reply:
Improved the error message to print all uninitialized fields.